Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -109,6 +109,13 @@ namespace Multiplayer
         "Should the server send all its individual multiplayer component version information to the client when there's a mismatch? "
         "Upon receiving the information, the client will print the mismatch information to the game log. "
         "Provided for debugging during development, but you may want to mark false for release builds.");
+    AZ_CVAR(
```

Comment:
if you want to ignore version mismatch (keep the connection), would the existing _sv_versionMismatch_autoDisconnect=false_ do the trick?

Reply:
That didn't work. The client still failed to connect.

Reply:
Actually, yeah, it does work but throws up an annoying warning on client connect. I wanted a debug switch that entirely disables it.